docs/HowTo/Configure/Consensus-Protocols/IBFT.md

```diff
@@ -25,12 +25,20 @@ Existing validators propose and vote to
 [add or remove validators](Add-Validators.md#ibft-20). Adding or removing a validator
 requires a majority vote (greater than 50%) of validators.
 
-## Minimum number of validators
+## Validators
+
+### Minimum number of validators
 
 IBFT 2.0 requires four validators to be Byzantine fault tolerant. Byzantine fault tolerance is the
 ability for a blockchain network to function correctly and reach consensus despite nodes failing or
 propagating incorrect information to peers.
 
+### Maximum number of validators
+
+As the number of validators increase, the message complexity increases as well, which can cause a loss of performance.
+In [network tests](https://wiki.hyperledger.org/display/BESU/Maximum+Validator+count+for+an+IBFT2+Network), IBFT 2.0 handles up to 30 validators safely with no loss of performance.
+After 35 validators, transaction pool time grows exponentially.
```

Comment:
Think we should just remove this last sentence "After 35 validators, transaction pool time grows exponentially.". We haven't finished analysis on why it's failing at this point.

Reply:
@jframe All set, could you re-review? Do you know if these limits also apply to QBFT?

Reply:
The first part "As the number of validators increase, the message complexity increases as well, which can cause a loss of performance." does apply to QBFT as well. But since we haven't done this network test for QBFT we can't say at what point QBFT stops functioning well though it's quite likely to be the same number.